docs/src/install.md

```diff
@@ -6,6 +6,32 @@ to install the latest stable version of this package:
 ```julia
 pkg> add Bloqade
 ```
+## Using Bloqade Optimized AMIs from AWS Marketplace
+The Bloqade team has implemented 2 dedicated AMIs that can be acquired using AWS Marketplace
+
+- Bloqade AMI with Julia
+- Bloqade Optimized Deep Learning AMI with CUDA and Julia
+
+
+### Bloqade AMI with Julia (base image)
+The Bloqade AMI contains:
+- The latest Julia installation using juliaup '1.5.1' which Julia '1.5.1'  for more information please refer to [Juliaup](https://github.com/JuliaLang/juliaup) 
```

Comment:
we should do 

```sh
juliaup add release
```

as default, since we always want the latest stable release.

Reply:
```suggestion
- The latest Julia installation using `juliaup add release` which setup the latest Julia release. For more information please refer to [Juliaup](https://github.com/JuliaLang/juliaup) 
```